SPECS/ksh/ksh.spec

```diff
@@ -80,12 +73,12 @@ touch %{buildroot}%{_bindir}/rksh
 touch %{buildroot}%{_mandir}/man1/rksh.1.gz
 
 %check
-# We run more tests as non-root user
-chmod g+w . -R
-useradd test -G root -m
-
-# Disabling tests as they tend to freez and the package is low-pri at the moment.
-false && sudo -u test ./bin/shtests --compile
+# disable pty tests as they tend to freeze
+shopt -s globstar; shopt -s nullglob; for test in bin/tests/**/*.sh; do
+  if [[ $test != *"pty.sh" ]]; then
+    $test || exit 1;
+  fi;
+done
```

Comment:
This looks a bit hacky. I think we're relying on finding and running the tests ourselves instead of the way upstream suggests running them. This is prone to break during version updates or we may miss a critical prep step for the tests.  And it looks like they do quite a few things inside `downloaded_sources/ksh-1.0.8/src/cmd/ksh93/tests/shtests`. Would looking for and removing the `pty.sh` work? It still won't resolve the potential break after a version upgrade but seems a bit more reliable and we can still run upstream's way of starting tests.